getting-started/deployment_guide.md

```diff
@@ -94,6 +96,69 @@ karmor log
 ## Prerequisites
 1. [K3s](../deployments/k3s)
 2. [MicroK8s](../contribution/microk8s)
-3. [Minikube](../contribution/minikube#minikube-installation)
+  <details>
+3. <summary>Minikube</summary>
+    If virtualbox and vagrant are not installed on your machine, please the following commands in advance.
```

Comment:
```suggestion
3. <details>
   <summary>Minikube</summary>
    If virtualbox and vagrant are not installed on your machine, please the following commands in advance.
```

This might help.